mlptrain/sampling/plumed.py

```diff
@@ -1033,6 +1033,59 @@ def __init__(self, name: str, atom_groups: Sequence = None):
         )
 
 
+class PlumedCNCV(_PlumedCV):
+    def __init__(
+        self,
+        name: str,
+        ref: float,
+        n: int,
+        m: int,
+        atom_groups: Sequence = None,
+    ):
+        """
+        PLUMED collective variable as a coordination number (CN) between two atoms or groups of atoms
+
+        e.g. [(0, 2)] gives ζ =[1-(r_02/r_ref)^6]/ [1-(r_02/r_ref)^12]
```

Comment:
The example should reflect the n, m.
```suggestion
        e.g. [(0, 2)] gives ζ =[1-(r_02/r_ref)^n]/ [1-(r_02/r_ref)^m]
```